e2e/TROUBLESHOOTING.md

```diff
@@ -5,10 +5,88 @@ The first step in determining the exact failure is to try and reproduce the test
 
 Each test suite's documentation should contain the specifics to run the tests, but it's summarises to executing `go test` or `godog` in the right directory.
 
-### SSH into the Cloud machines
-On CI, we are running the Elastic Stack and all test suites in AWS instances, so whenever a build failed we would need to access those machines and inspect the state of the machine: logs, files, containers... For that, we are enabling SSH access to those ephemeral machines, which will be kept for debugging purpose if and only if the DEVELOPER_MODE environment variable is set at the Jenkinsfile. In the UI of Jenkins, you can enable it using the DEVELOPMENT_MODE input argument, checking it to true (default is false). After the build finishes, the cloud instances won't be destroyed.
+### Running the tests on the Cloud machines
+On CI, we are running the Elastic Stack and all test suites in AWS instances, so whenever a build failed we would need to access those machines and inspect the state of the machine: logs, files, containers... For that, we are enabling SSH access to those ephemeral machines, which will be kept for debugging purpose if and only if the `DEVELOPER_MODE` environment variable is set at the Jenkinsfile. In the UI of Jenkins, you can enable it using the `DEVELOPER_MODE` input argument, checking it to true (default is false). After the build finishes, the cloud instances won't be destroyed.
 
-To access the machines, you must be allowed to do so first, and for that, please submit a PR adding your Github username in alphabetical order to [this file](../.ci/ansible/github-ssh-keys), keeping a blank line as file ending. The user to access each EC2 used on the tests can be found [here](https://github.com/elastic/e2e-testing/blob/main/.ci/.e2e-platforms.yaml).
+But you must first understand that there are two types of Cloud machines: 1) the VM running the stack, and 2) the VMs where the Elastic Agent will be installed and enrolled into the stack.
```

Comment:
[Suggestion]
It's in part personal opinion, but I find it easier to read when the items are each one on a separated line:

```suggestion
But you must first understand that there are two types of Cloud machines: 
1) the VM running the stack, and 
2) the VMs where the Elastic Agent will be installed and enrolled into the stack.
```